README.md

```diff
@@ -4,7 +4,29 @@
 [![Build
 Status](https://action-badges.now.sh/open-telemetry/opentelemetry-dotnet-contrib)](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions)
 
+This repository contains set of components extending functionality of the
+OpenTelemetry SDK. Instrumentation libraries, exporters, and other components
+can find its home here.
+
+Packages in this repository has a prefix `OpenTelemetry.Contrib` to highlight
+the difference from the packages produced by the primary .NET SDK repository.
+
 ## Contributing
 
 For information on how to contribute, consult [the contributing
 guidelines](./CONTRIBUTING.md)
+
+## Support
+
+This repository is maintained by [.NET SDK
+maintainers](https://github.com/orgs/open-telemetry/teams/dotnet-maintainers)
+team and [.NET SDK
+approvers](https://github.com/orgs/open-telemetry/teams/dotnet-approvers) can
+help with reviews and code approval. However, as individual components are
+developed by numerous contributors, approvers and maintainers are not expected
+to directly contribute to every component.
+
+This repository is great for community supported components. Vendor specific
+code that deliver a higher supportability guarantees needs to be placed in
```

Comment:
Something about `deliver a higher supportability guarantees` seems a bit off.

`requires a higher supportability guarantee`

Maybe?

Reply:
fixed

Reply:
I think it should also be `guarantee` not `guarantees`?